template-parts/content/error-404.php

```diff
@@ -19,6 +19,7 @@
 		<?php
 		get_search_form();
 
+		wp_print_styles( array( 'wp-rig-widgets' ) );
```

Comment:
Function call needs updating to `wp_rig()->print_styles()` per #319.

Reply:
Updated. Thank you for the review.